### ISSUE TITLE
Ideal value of global scale var Struct2depth

### ISSUE THREAD

**Issue by ezorfa** [tensorflow/models#8046]
Hi @aneliaangelova  !

Could you please let me know what should be the ideal value of global scale var, when its trained?

I  am trying to implement "object constraint loss" in pytorch but facing difficulties. The scale var I defined slowly decreases to zero. I posted a similar question on stackoverflow : https://stackoverflow.com/questions/59800247/pytorch-equivalent-of-tf-variable

Thankyou!

What is the top-level directory of the model you are using : Custom
Have I written custom code : yes, In pytorch
OS Platform and Distribution : Linux 16.04
TensorFlow installed from: None
TensorFlow version: None
Bazel version: None
CUDA/cuDNN version: 10
GPU model and memory: GeForce 11GB
Exact command to reproduce : None

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
